### PR TITLE
Calculator:  patch overtriggering

### PR DESCRIPTION
**Pull request by pjhampton**
## Description of new Instant Answer, or changes

Patches some overtriggering.

## People to notify

@moollaza 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Code**
- [x] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [x] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [x] There is no unnecessary files in place (such as editor config files)
- [x] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/testing-reference/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/calculator

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: t/Calculator.t [hunk 1]
```diff
@@ -535,7 +535,11 @@ ddg_goodie_test(
     '.elxs' => undef,
     '.gif' => undef,
     'e.png' => undef,
+    '(929) 665-83-03' => undef,
+    '(343) 270-55-66' => undef,
+    '2014 E350' => undef,
     ')1108278829' => undef,
+    '24score' => undef,
```
Comment (mintsoft) [inline t/Calculator.t]: how about `24 score` too?

File: t/Calculator.t [hunk 2]
```diff
@@ -535,7 +535,11 @@ ddg_goodie_test(
     '.elxs' => undef,
     '.gif' => undef,
     'e.png' => undef,
+    '(929) 665-83-03' => undef,
+    '(343) 270-55-66' => undef,
+    '2014 E350' => undef,
```
Comment (mintsoft) [inline t/Calculator.t]: There is also some overlap for 199x, like the 1997 ford E350 or the 1994 E34 (a BMW)

Comment (moollaza) [inline t/Calculator.t]: Fixed. Looks like BMW goes back to the 80's 👍

**Comment by moollaza** [comment]
@pjhampton Thanks! This LGTM :+1:

**Comment by bsstoner** [comment]
This was released 🚀

**Pull request merged**